### ISSUE TITLE
Grep, replace do not work at the Vault root

### ISSUE THREAD

**Issue by mattlqx** [fishi0x01/vsh#93]
The most broad you can use these commands today is that the top of a KV store. You can't use them at the "root" of a server to search all KV stores. It would be nice if an entire Vault can be searched rather than having to specify every KV store.

```
https://localhost:8200 /> grep foo ./
Not a valid path for operation: //
https://localhost:8200 /> 
```

**Comment by fishi0x01** [comment]
Good point. Im refactoring to uniform the traversal functions for concurrency https://github.com/fishi0x01/vsh/pull/86
I think with that I can also fix this issue.

Didn't have much time lately to work on this. But will try to get to it soon again :)
